ariba/2.14.6/README.md

```diff
@@ -0,0 +1,38 @@
+# ARIBA container
+
+GitHub Repo: [ARIBA](https://github.com/sanger-pathogens/ariba)
+
+Full documentation: [https://github.com/sanger-pathogens/ariba/wiki](https://github.com/sanger-pathogens/ariba/wiki)
+
```

Comment:
Can you add a section that lists the additional tools that were installed and their versions? I think it's just spades and pysam. An example README for comparison can be found at https://github.com/StaPH-B/docker-builds/tree/master/dnaapler/0.1.0